workers/unity/Assets/Gdk/Core/Components/WorldCommandsTranslation.cs

```diff
@@ -1,36 +1,38 @@
 using System.Collections.Generic;
 using Improbable.Gdk.Core.Components;
 using Improbable.Worker;
+using Improbable.Worker.Query;
+using Tests;
```

Comment:
Also what's up with this import?

Reply:
Sorry, used to IntelliJ auto cleaning imports and committed in a hurry. Will remove.